### ISSUE TITLE
Notes are in iCloud but are not loaded in FSNotes

### ISSUE THREAD

**Issue by oreglio** [glushchenko/fsnotes#1085]
**Describe the bug**
I opened FSNotes on MacOS BigSur without any system update or any other update. FSNotes is opening like it's the first time,  without any notes. But when I open it in my iPhone it's all there and it's there as well in iCloud > FSNotes in my Mac.

**To Reproduce**
I didn't do anything, no MacOS Update, I just use FSNotes from the Mac App Store so it's being updated automatically. I haven't opened FSNotes on the Mac for a few weeks, there might have been several updates maybe ? I don't check updates because they are automatic.

**Expected behavior**
All the notes from iCloud > FSNotes to load

**Actual behavior**
All the notes from iCloud > FSNotes to load are present in the directory but not loaded in FSNotes for Mac. 

**Workaround**
I use FSNotes in iOS to get some notes for now but it's not practical.

**Desktop (please complete the following information):**
 - OS: macOS Big Sur 
 - OS version: 11.3 (20E5172i)

 - FSNotes version: 4.9.8 buid 462

**Additional context**
In the second screenshot under "General" in clicked on the "change" button to capture that there is all my notes there and when I clicked on "open", without changing the path or anything, it asked me if I wanted to change the path for the notes and I clicked on "no". And now I have what appears in the last screenshot: some directories are there but I have a 1e14 - FSNotes.git in the directory  iCloud > FSNotes ... ! These appear now on my iOS FSNotes app, it's kind of a nightmare.

**Supporting Files**
<img width="1216" alt="Capture d’écran 2021-03-05 à 09 51 05" src="https://user-images.githubusercontent.com/537638/110091059-546f1880-7d98-11eb-976b-3c74252f48ec.png">
<img width="1208" alt="Capture d’écran 2021-03-05 à 09 55 12" src="https://user-images.githubusercontent.com/537638/110091781-1f16fa80-7d99-11eb-80ac-21b6263e854f.png">
<img width="1210" alt="Capture d’écran 2021-03-05 à 09 59 08" src="https://user-images.githubusercontent.com/537638/110092433-e592bf00-7d99-11eb-98d9-ebbb915b9784.png">
![Image PNG](https://user-images.githubusercontent.com/537638/110092620-1115a980-7d9a-11eb-97e0-45c249b721e3.png)

**Comment by glushchenko** [comment]
Archive .git repo in iCloud Drive for what? Surely you broke iCloud Drive.

**Comment by oreglio** [comment]
That is what I'm trying to do, I NEVER changed anything and the .git repo is NOT mine !

**Comment by oreglio** [comment]
I trusted FSNotes with all my data, I bought it and the only answer here is closing my issue ?

I'm pulling the plug on FSNotes it's really not reliable.

**Comment by glushchenko** [comment]
![image](https://user-images.githubusercontent.com/1089917/110096994-34932100-7da7-11eb-8ac9-15ce8373b4ee.png)

> I trusted FSNotes with all my data, I bought it and the only answer here is closing my issue ?

Yes.

**Comment by oreglio** [comment]
That is the DEFAULT path I never changed anything so with my eyes with open can you enlight me about what is wrong with Archive being the archive storage ?

**Comment by glushchenko** [comment]
Please open tab Git and check repositories path. You should not store git archive in iCloud Drive.

**Comment by oreglio** [comment]
Beyond that I you read the issue from start to bottom I came with a problem and went down with two problems.
You completely ignored the first problem that is: at first there wasn't any .git repo, just all my notes present on iCloud > FSnotes and loading correctly on iOS but not loading on the Mac.
Then I had with .git repo error while doing screenshots for the issue

**Comment by glushchenko** [comment]
Really? What this?

![image](https://user-images.githubusercontent.com/1089917/110099090-96ed2100-7da9-11eb-937f-63269606036c.png)

**Comment by glushchenko** [comment]
You are storing git archive in the cloud and this is the problem. If you are not satisfied, request a refund.

**Comment by oreglio** [comment]
I'm telling you that I came here WITHOUT the .git problem and FSNotes was NOT loading the notes in the Mac app. And WHILE writing the issue this .git problem appeared. So there is a real issue because I haven't used FSNotes for weeks and when I opened it there wasn't this .git problem. And there wasn't and .git repo in the iOS app. It's when I did the screenshots that the .git repo appeared here.

Seriously I'm really disappointed here not only I have no real solution to my problem but I feel like you're not taking my issue seriously but giving the advice of asking for a refund ? Really ?

I took the time to document the issue to have some help and that is how to take it ?

Let me burn alone if it's how you do things.

Thank you for your time.
